dataplane/vppagent/scripts/vpp_listener.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
```

Comment:
Why not Go?

Reply:
It is derived from the original [supervisord_kill.py](https://github.com/ligato/vpp-agent/blob/master/docker/prod/supervisord_kill.py). I don't see a need to rewrite it in go here.